proposals/0029-cooperative-vector.md

```diff
@@ -602,92 +609,90 @@ The vector-matrix intrinsics are expected to be supported in all shader stages.
 
 #### D3D12 API Additions
 
-Note: The enums and structs need to be updated from the coop_vec name, once a
-new name for the feature is decided.
-
 ### Check Feature Support
 
 This feature requires calling CheckFeatureSupport(). Additional D3D12_FEATURE
 enum and corresponding D3D12_FEATURE_DATA* structs (listed below) are added to
-enable discovering the Cooperative Vector tier along with the datatype and
+enable discovering the Linear Algebra: Matrix Vector tier along with the datatype and
```

Comment:
If we plan to add Matrix-Matrix operations under the same feature in the future, under a higher tier, can we just make the **Linear Algebra** versus **Linear Algebra Matrix Vector**

Reply:
The problem is what if an implementation can only support the base tier of MatVec and base tier of MatMat?  Would need different tiers.  Or is there no chance this would happen and we're sure MatMat ops can be required to layer on all MatVec stuff being supported?